build-system/tasks/extension-helpers.js

```diff
@@ -487,7 +485,7 @@ async function buildExtension(
 
   await compileJison(path.join(extDir, '**', '*.jison'));
   if (name === 'amp-bind') {
-    await doBuildJs(jsBundles, 'ww.max.js', options);
+    await doBuildJs(jsBundles, 'ww.max.js', {...options, watch: false});
```

Comment:
To clarify, I assume this change means that during first-time build, this call will explicitly build `ww.js` along with `amp-bind`, but during subsequent watch builds, the watcher will automatically build both `amp-bind` and `ww.js`? (This is cool!)

Reply:
Yes! Mostly! Sort of. When building amp-bind initially, it will build both but only initiate watching for `amp-bind` direct dependencies (as opposed to the binary dependency on ww).

This mitigates the double-watch that would have been created by the lazy server when requesting ww.max.js.

There are still a few situations that cause a double build, so this isn't ideal:
1. First time lazy-requesting the bundle will cause a double build
2. Modifying the jison/css within amp-bind

Reply:
It would potentially be better to have `compileJs` silently ignore any 2nd request to watch the same entrypoint

Reply:
Opted for the smarter strategy. `compileJs` is now happy being called N times with `watch: true` and it will behave intelligently (no double watch etc)